returnn/frontend_api.py

```diff
@@ -42,6 +42,20 @@ def convert_to_tensor(value: Union[Tensor, T, RawTensorTypes]) -> Tensor[T]:
         """
         raise NotImplementedError
 
+    @staticmethod
+    def _tensor_op_sanity_check(t1: Union[RawTensorTypes, Tensor], t2: Union[RawTensorTypes, Tensor]):
+        """
+        Performs basic checks common to all _TensorMixin binary operations,
+        like _TensorMixin.__eq__ or _TensorMixin.__add__.
+        :param t1:
+        :param t2:
+        """
+        if isinstance(t1, _t.Tensor) and isinstance(t2, _t.Tensor):
```

Comment:
The way you call `_tensor_op_sanity_check`, you always expect that you have `isinstance(t1, _t.Tensor) and isinstance(t2, _t.Tensor)`, right? Then don't make this an `if` check, but rather an `assert`.